src/containers/Registration.tsx

```diff
@@ -0,0 +1,186 @@
+import { yupResolver } from "@hookform/resolvers/yup";
+import { useState } from "react";
+import { Button, Form, FloatingLabel } from "react-bootstrap";
+import { useForm } from "react-hook-form";
+import styled, { css } from "styled-components";
+import * as yup from "yup";
+
+export const windowTitle = "Створіть свій профіль";
+
+const StyledForm = styled(Form)`
+  padding: 0 2rem;
+`;
+
+const StyledFormControl = styled(Form.Control)`
+  margin-top: 1rem;
+  padding: 2rem 0.75rem 2rem 1.7rem !important;
+  height: 5.5rem !important;
+  font-size: 1.5rem;
+
+  ${(props) =>
+    props.top &&
+    css`
+      margin-top: 0rem;
+    `}
+`;
+
+const StyledFloatingLabel = styled(FloatingLabel)`
+  font-size: 1.5rem;
+  label {
+    padding: 2rem 1.7rem;
+  }
+  .form-floating > .form-control:focus ~ label,
+  .form-control:not(:placeholder-shown) ~ label,
+  .form-control:focus ~ label {
+    transform: scale(0.8) translateY(-1.5rem) translateX(0);
+    opacity: 0.65 !important;
+  }
+`;
+
+const StyledCounterLabel = styled.label`
+  text-align: right;
+  opacity: 0;
+  padding-right: 0;
+  margin-right: 0;
+  min-width: 125%;
+  transform-origin: 0 0;
+`;
+
+const StyledButton = styled(Button)`
+  margin-top: 1rem;
+  border-radius: 9999px;
+  padding: 1rem;
+`;
+
+interface MyForm {
+  userName: string;
+  email: string;
+  dateOfBirth: string;
+  password: string;
+  confirmPassword: string;
+}
+
+const schema = yup.object().shape({
+  userName: yup.string().required("Як вас звати?"),
```

Comment:
maybe move the signature to the right

Reply:
What is signature? All code formatted with Prettier.